dipy/tracking/streamline.py

```diff
@@ -113,3 +116,169 @@ def select_random_set_of_streamlines(streamlines, select):
     len_s = len(streamlines)
     index = np.random.randint(0, len_s, min(select, len_s))
     return [streamlines[i] for i in index]
+
+
+def reduce_rois(rois, include):
```

Comment:
OK - moved `reduce_rois` to utils.

Seems logical to have `select_by_rois` in `streamline`, right next to `select_random_set_of_streamlines` (and that's also the comment you made here on July 20th), and it makes sense to have `streamline_near_roi` in `utils`, together with `target`. I would not replace `target` with that other function, considering that they do slightly different things, and have slightly different use-cases.